### DISCUSSION TITLE
DB2/400 modify column properties in a table

### DISCUSSION THREAD

**Discussion by MarioMoleri** [dbeaver/dbeaver#14212] [General]
Hello everyone,

DBeaver Community Edition	21.2.2.202110030714	.

Someone can be so kind to let me know why I can't modify column properties in a table on db2/400 connection ?
some properties to set in connection ?

Thanks a lot in advance.
Regards

**Comment by LonwoLonwo** [comment]
Hello @MarioMoleri 

That's because this feature is not implemented. 
You can create a ticket as a feature request here: https://github.com/dbeaver/dbeaver/issues

But I'm afraid, that we can't do this in the near future because we do not have a test environment.

**Reply by MarioMoleri** [reply]
Hello @LonwoLonwo ,

Thanks a lot for your answer.
Could be a great feature.

For testing environment we can find a solution, and collaborate.

**Reply by LonwoLonwo** [reply]
> For testing environment we can find a solution, and collaborate.

It can be very helpful for us!

You can send me more info at anvolka@jkiss.org
